plugins/spam.py

```diff
@@ -12,76 +12,87 @@
                    "Btw Don't over use this plugin or get"
                    "ready for account ban or flood waits. "
                    "For spamming text use '|' to separate count and text.",
-    'usage': "{tr}spam [spam count] | [spam message/reply to a media]",
+    'usage': "{tr}spam [spam count] | [spam message/reply to a media] | [Optional: delay]",
     'examples': "**For Text:** `{tr}spam 2 | Durov will ban me for using this plugin`"})
 async def spam(message: Message):
     replied = message.reply_to_message
+    delay = 0.1
```

Comment:
```python
delay = "0.1"
```
???

because you are checking `'.' in delay`

Reply:
wen giving some Proness peru sir